runtime/src/bank.rs

```diff
@@ -6060,9 +6079,23 @@ impl Bank {
                 }
                 total_rent_collected_info += rent_collected_info;
                 accounts_to_store.push((pubkey, account));
+            } else if test_skip_rewrites_but_include_hash_in_bank_hash {
+                // include rewrites that we skipped in the accounts delta hash.
+                // This is what consensus requires prior to activation of bank_hash_skips_rent_rewrites.
+                // This code path exists to allow us to test the long term effects on validators when the skipped rewrites
+                // feature is enabled.
+                let hash = AccountsDb::hash_account(account, pubkey);
+                skipped_rewrites.push((*pubkey, hash));
             }
             rent_debits.insert(pubkey, rent_collected_info.rent_amount, account.lamports());
         }
+        if test_skip_rewrites_but_include_hash_in_bank_hash {
+            // these account pubkeys will have to be added to the bank hash's accounts delta hash
+            self.skipped_rewrites
+                .write()
+                .unwrap()
+                .append(&mut skipped_rewrites);
```

Comment:
The downside is that it will change the function signature and the caller will need updates. Since this is the test feature, not sure if it is worthwhile?

Reply:
updated.